content/en/docs/reference/config-api/apiserver-config.v1.md

```diff
@@ -38,6 +39,96 @@ auto_generated: true
 </tbody>
 </table>
 
+## `EncryptionConfiguration`     {#apiserver-config-k8s-io-v1-EncryptionConfiguration}
+    
+
+
+<p>EncryptionConfiguration stores the complete configuration for encryption providers.
+It also allows the use of wildcards to specify the resources that should be encrypted.
+Use '<em>.<!-- raw HTML omitted -->' to encrypt all resources within a group or '</em>.<em>' to encrypt all resources.
```

Comment:
This looks a bit odd; is there anything easy we can do about it?

Reply:
Ah yes. The original code comment was written in a way that confuses the markdown parser. We have to manually fix it. Will do.